source/clear-linux/tutorials/kata.rst

```diff
@@ -97,10 +87,32 @@ environment:
 You do not need to manually configure the runtime for Docker, because
 it automatically uses the runtime supported by the system.
 
-Check which runtime your system is using with the command:
+Troubleshooting
+===============
+
+To check which runtime your system uses, enter:
 
 .. code-block:: bash
 
    sudo docker info | grep runtime
 
+For some |CL| versions before 27000, you may need to manually
+configure Docker\* to use Kata Containers by default.
+
+To do so, enter:
+
+.. code-block:: bash
+
+   sudo mkdir -p /etc/systemd/system/docker.service.d/
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/51-runtime.conf
+   [Service]
+   Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime kata-runtime"
+   EOF
+
```

Comment:
@eadamsintel. Why does the older version use `51-runtime.conf`, but the new version uses `50-runtime.conf`? 
Should this older version be corrected to `...50-runtime.conf`?

Reply:
Sure.  You can manually create any version you want.  I'd make it consistent.

Reply:
Agreed. I'll make them both `...50-runtime.conf`